### PR TITLE
Handle directories

### LINKED ISSUE #14: Bug with matching paths within .gitignore directory
Is this correct? It feels like a bug to me since it works when you specify the rule in `.gitignore`.

Imagine `.gitignore` contents including

```
.venv/
```

And a script evaluating this

```
gitignore_matcher('/path/to/repo/.venv/')
True
gitignore_matcher('/path/to/repo/.venv/bin')
False
```

To me the second one should return `True`

### PR DESCRIPTION
**Pull request by inverse**
Starting to fix #14

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -19,6 +20,18 @@ def test_simple(self):
 		self.assertTrue(matches('/home/michael/dir/main.pyc'))
 		self.assertTrue(matches('/home/michael/__pycache__'))
 
+	def test_ignore_directory(self):
+		matches = _parse_gitignore_string('.venv/', fake_base_dir='/home/michael')
+		self.assertTrue(matches('/home/michael/.venv'))
+		self.assertTrue(matches('/home/michael/.venv/folder'))
+		self.assertTrue(matches('/home/michael/.venv/file.txt'))
+
+	def test_ignore_directory_astrix(self):
```
Comment (inverse) [inline tests.py:29]: Currently failing

**Comment by bitranox** [comment]
@inverse, pulled in Your changes on version 1.0.2 of igittigitt : https://github.com/bitranox/igittigitt

**Comment by mherrmann** [comment]
Thank you @inverse. Could you resolve the conflicts so I can merge? Then I can release a new version with those improvements to PyPI.

**Comment by inverse** [comment]
@mherrmann resolved conflicts :+1:

**Pull request merged**